src/ui/UI.java

```diff
@@ -135,6 +135,7 @@ private void getUserCredentials() {
 				repoSelector.setDisable(false);
 				repoSelector.refreshComboBoxContents(ServiceManager.getInstance().getRepoId().generateId());
 				triggerEvent(new BoardSavedEvent());
+				getMenuControl().scrollTo(columns.getCurrentlySelectedColumn().get(), columns.getChildren().size());
```

Comment:
There may not be a selected column at this point. You should check `columns.getCurrentlySelectedColumn().isPresent()`, or assert it.